src/test/java/com/fasterxml/jackson/dataformat/xml/ser/XmlPrettyPrinterTest.java

```diff
@@ -91,8 +91,6 @@ public void setUp() throws Exception {
         _xmlMapper.configure(SerializationFeature.INDENT_OUTPUT, true);
     }
 
-    private static final String SYSTEM_DEFAULT_NEW_LINE = System.getProperty("line.separator");
-
```

Comment:
There was dynamically set `SYSTEM_DEFAULT_NEW_LINE` in ONLY `XmlPrettyPrinterTest.java`.
I removed it and moved it up to XmlTestBase.